app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -37,6 +39,10 @@ def email_from_sp_identity(identity)
     EmailContext.new(identity.user).last_sign_in_email_address.email
   end
 
+  def alternate_emails_from_sp_identity(identity)
+    EmailContext.new(identity.user).alternate_email_addresses.map(&:email)
+  end
+
```

Comment:
we use `EmailContext.new(...)` up above, so let's memoize out a method? save some allocations + queries

```suggestion
  def alternate_emails_from_sp_identity(identity)
    email_context.alternate_email_addresses.map(&:email)
  end

  def email_context
    @email_context ||= EmailContext.new(identity.user)
  end
```